src/gradcheck.jl

```diff
@@ -142,7 +142,7 @@ function gc_scalar(f)
             # srand(r,1)
             # a = oftype(v, rand(r, size(v)))
             # return sum(y .* a)
-            if isa(getval(y), Associative)
+            if isa(getval(y), AbstractDict)
```

Comment:
Where does `AbstractDict` come from in Julia v0.6? I couldn't have found it.

Reply:
It is provided by `Compat`. I am running this against `Compat 0.63` - I updated `REQUIRE` to require that version.